### PR TITLE
[APR] Add datepicker

### PR DESCRIPTION
**Pull request by Rawallon**
Adds date picker component

https://github.com/bleu-fi/balancer-tools/assets/68453900/3b836523-3ea3-4757-a56a-4cd73d471235

### CODE REVIEW

File: apps/balancer-tools/src/components/Accordion.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ export const AccordionTrigger = forwardRef<
     >
       {children}
       <ChevronDownIcon
-        className="text-white transition-transform duration-300 ease-[cubic-bezier(0.87,_0,_0.13,_1)] group-data-[state=open]:rotate-180"
+        className="text-white transition-transform duration-300 ease-&lsqb;cubic-bezier(0.87,_0,_0.13,_1)&rsqb; group-data-[state=open]:rotate-180"
```
Comment (ribeirojose) [inline apps/balancer-tools/src/components/Accordion.tsx:34]: 👀 why?

File: apps/balancer-tools/src/components/ui/date-range-picker.tsx [hunk 2]
```diff
@@ -0,0 +1,549 @@
...
+            Update
+          </Button>
+        </div>
+      </PopoverContent>
+    </Popover>
+  );
+};
+
+DateRangePicker.displayName = "DateRangePicker";
+DateRangePicker.filePath =
```
Comment (ribeirojose) [inline apps/balancer-tools/src/components/ui/date-range-picker.tsx:548]: 👀

File: apps/balancer-tools/src/components/ui/button.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import { Slot } from "@radix-ui/react-slot";
```
Comment (ribeirojose) [inline apps/balancer-tools/src/components/ui/button.tsx:1]: 👍 cool implementation, but can you merge this into our already existing `Button` component?

File: apps/balancer-tools/components.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (ribeirojose) [inline apps/balancer-tools/components.json:1]: what do we need this file for?

Comment (Rawallon) [reply apps/balancer-tools/components.json:1]: This is created on shadcn's init

File: apps/balancer-tools/src/components/ui/button.tsx (hunk 3 above)
Comment (Rawallon) [reply apps/balancer-tools/src/components/ui/button.tsx:1]: Hm, didn't realize it was in the `ui` folder, I might move them to components

File: apps/balancer-tools/components.json (hunk 4 above)
Comment (luizakp) [reply apps/balancer-tools/components.json:1]: https://ui.shadcn.com/docs/installation/next

File: apps/balancer-tools/src/components/ui/calendar.tsx
Comment (luizakp) [inline apps/balancer-tools/src/components/ui/calendar.tsx:1]: this entire file was generated by `npx shadcn-ui@latest add calendar`?

File: apps/balancer-tools/src/components/Accordion.tsx (hunk 1 above)
Comment (luizakp) [reply apps/balancer-tools/src/components/Accordion.tsx]: reversed it

File: apps/balancer-tools/src/components/ui/date-range-picker.tsx (hunk 2 above)
Comment (luizakp) [reply apps/balancer-tools/src/components/ui/date-range-picker.tsx:543]: think this is also generated by shadcn when we run `npx shadcn-ui@latest add date-range-picker`

File: apps/balancer-tools/components.json (hunk 4 above)
Comment (ribeirojose) [reply apps/balancer-tools/components.json:1]: that's a new one for me, I'd just copy the components themselves 😅

File: apps/balancer-tools/src/components/ui/date-range-picker.tsx (hunk 2 above)
Comment (ribeirojose) [reply apps/balancer-tools/src/components/ui/date-range-picker.tsx:543]: ok, but why keep it here? Wonder if this is a bug on their side. It doesn't look like it's the sort of thing that should be here.

File: apps/balancer-tools/src/components/ui/calendar.tsx
Comment (Rawallon) [reply apps/balancer-tools/src/components/ui/calendar.tsx:1]: Yup

**Pull request merged**